src/components/package/show/package-show.js

```diff
@@ -81,14 +87,31 @@ class PackageShow extends Component {
     this.setState({ isCoverageEditable });
   };
 
+  handleSectionToggle = ({ id }) => {
+    this.setState((curState) => {
```

Comment:
Instead of cloning the object with deep copy, we could use `lodash/fp/update`. `fp` stands for functional programming. It's unfortunately not well documented :( but it allows to do a set as an immutable operation. 

I made an example of you of how it can be used https://runkit.com/taras/5b606de43fb58c001196f7a3

You should be able to replace this code with,

```js
let next = update(`sections.${id}`, value => !value, this.state);

this.setState(next);
```

Here is the documentation for [update function](https://gist.github.com/jfmengels/6b973b69c491375117dc#_updatepath-updater-object).